.travis.yml

```diff
@@ -11,6 +11,9 @@ jobs:
   include:
     # Define the release stage that runs semantic-release
     - stage: release
+      node_js:
```

Comment:
Since we still support Node 8, shouldn't we run it on 8 as well? @anescobar1991 thoughts?

Reply:
We would still be testing on the node 8 and only using semantic-release which is a dev dependency on node 10. https://github.com/semantic-release/semantic-release/blob/master/docs/support/node-version.md#node-version-requirement

Reply:
Got it 👍